db/re/item_db_usable.yml

```
@@ -73679,8 +73679,8 @@ Body:
     Flags:
       BuyingStore: true
     Script: |
-      /* TODO : pet 21632; */
-      pet 1005;
+      /* TODO : pet; */
```

```suggestion
      /* TODO : pet 21632; */
```
Otherwise the important information may be lost.